python/semantic_kernel/connectors/mcp.py

```diff
@@ -102,7 +102,7 @@ def get_parameter_from_mcp_prompt(prompt: Prompt) -> list[dict[str, Any]]:
 def get_parameters_from_mcp_tool(tool: Tool) -> list[dict[str, Any]]:
     """Creates an MCPFunction instance from a tool."""
     properties = tool.inputSchema.get("properties", None)
-    required = tool.inputSchema.get("required", None)
+    required = tool.inputSchema.get("required", list())
```

Comment:
```suggestion
    required = tool.inputSchema.get("required", [])
```
see https://stackoverflow.com/questions/5790860/whats-the-difference-between-and-vs-list-and-dict